s3.js

```diff
@@ -126,6 +126,25 @@ S3Storage.prototype.del = function (key, opts, cb) {
   })
 }
 
+S3Storage.prototype.mdel = function (objects, cb) {
```

Comment:
cb should be optional like above

Reply:
I see that it is optional in some methods, e.g. `put` and `get`. But it doesn't seem to be optional in `del` or `exists`.
What determines if the callback ought to be optional or not?

Reply:
Ah that's a mistake. Should be optional in any op that doesnt return anything to the cb other than a potential error